components/form-category-input/category-input.styled.js

```diff
@@ -6,5 +6,6 @@ export const StyledCategoryView = styled.div`
   > * {
     color: whitesmoke;
     font-weight: bold;
+    position: relative;
   }
```

Comment:
Although it is not part of the PR, I would like to mention that this is kind of funky 😬.
I suggest splitting the child styles into another component (something like this not tested example below) but it's up to you.

```

 const StyledCategoryItem = styled.div`
    color: whitesmoke;
    font-weight: bold;
    position: relative;
`;

const StyledCategoryView = styled.div`
  width: 100%;
  height: 3em;
  border: 2px solid #2490b5;

  ${StyledCategoryItem} {
    color: whitesmoke;
    font-weight: bold;
    position: relative;
  }
`;
```

Reply:
as in the form the child does not need to be styled, i deleted this. btw. to avoid problematic prop-drilling, i took the categories out from their own component and implemented them to new-pedal-form.